packages/mobile/src/components/add-funds-drawer/AddCashDrawer.tsx

```diff
@@ -55,15 +55,18 @@ export const AddCashDrawer = () => {
   const onContinuePress = useCallback(() => {
     if (selectedPurchaseMethod === PurchaseMethod.CARD) {
       openCardFlow()
+      handleClose()
     } else if (selectedPurchaseMethod === PurchaseMethod.CRYPTO) {
       setPage(AddCashDrawerPage.TRANSFER)
     }
-  }, [selectedPurchaseMethod, openCardFlow])
+  }, [selectedPurchaseMethod, openCardFlow, handleClose])
 
   const handleClose = useCallback(() => {
     dispatch(resetPurchaseMethod())
+    setPage(AddCashDrawerPage.MAIN)
+    setSelectedPurchaseMethod(PurchaseMethod.CARD)
     onClose()
-  }, [dispatch, onClose])
+  }, [dispatch, onClose, setPage, setSelectedPurchaseMethod])
```

Comment:
Note @raymondjacobson the changes in these files are actually not needed. Dropping them